pkg/webhooks/jobset_webhook.go

```diff
@@ -53,26 +61,66 @@ const (
 	// Error message returned by JobSet validation if the network subdomain
 	// will be longer than 63 characters.
 	subdomainTooLongErrMsg = ".spec.network.subdomain is too long, must be less than 63 characters"
+
+	// k8sMinorForHardcodedBuitins is used for creating the OpenAPI client.
+	k8sMinorForHardcodedBuitins = "1.27"
```

Comment:
We are still supporting 1.26 until 1.30 is released right?

Reply:
I'm wrong. https://kubernetes.io/releases/#release-v1-26

1.26 was dropped as of last month. 

Not sure what your LTS support will be for GKE on this. but I guess its fine from upstream perspective.

Reply:
@danielvegamyhre @ahg-g WDYT of this?

Reply:
I think we should drop support for 1.26 (remove 1.26 e2e test) and keep 1.27 for the hardcoded builtins here

Reply:
Resolving since we dropped the 1.26 test